scripts/lib/upgrade-zulip-stage-2

```diff
@@ -206,16 +206,26 @@ elif args.from_git:
     # Ideally, we'd have 2 thresholds here, depending on whether the
     # system is running queue workers multithreaded or multiprocess.
     # See puppet/zulip/manifests/app_frontend_base.pp for background.
-    if mem_gib < 3.8:
-        logging.info("Shutting down server to build static assets on a low-RAM system.")
+    if mem_gib < 4.2:
+        logging.info("Shutting down server to ensure sufficient free RAM for webpack.")
         shutdown_server()
 
     # Note: The fact that this is before we apply Puppet changes means
     # that we don't support adding new Puppet dependencies of
     # update-prod-static with the Git upgrade process.  But it'll fail
     # safely; this seems like a worthwhile tradeoff to minimize downtime.
     logging.info("Building static assets...")
-    subprocess.check_call(["./tools/update-prod-static"], preexec_fn=su_to_zulip)
+    try:
+        subprocess.check_call(["./tools/update-prod-static"], preexec_fn=su_to_zulip)
+    except subprocess.CalledProcessError as e:
+        logging.exception(e, stack_info=True)
```

Comment:
Is the stack info actually useful here?

Reply:
I guess not, from looking at the above; we can cut that; `update-prod-static` has its own error output that's useful.